src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,279 @@
+
+        // Set price feed for RWA-00x
+        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
+
+        // Init RWA-00x in Vat
+        Initializable(vat).init(collateral.ilk);
+        // Init RWA-00x in Jug
+        Initializable(DssExecLib.jug()).init(collateral.ilk);
+
+        // Allow RWA-00x Join to modify Vat registry
+        DssExecLib.authorize(vat, collateral.MCD_JOIN);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
+
+        // set ilk/global DC
+        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
+
+        // No dust
+        // DssExecLib.setIlkMinVaultAmount(collateral.ilk, 0);
+
+        // stability fee
+        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, false);
+
+        // collateralization ratio
+        DssExecLib.setIlkLiquidationRatio(collateral.ilk, collateral.MAT);
+
+        // poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(collateral.ilk);
+
+        // give the urn permissions on the join adapter
+        // DssExecLib.authorize(collateral.MCD_JOIN, collateral.URN);
+
+        // set up the urn
+        Hopeable(collateral.URN).hope(collateral.OPERATOR);
+
+        // set up output conduit
+        // Hopeable(collateral.OUTPUT_CONDUIT).hope(collateral.OPERATOR));
+
+        // Authorize the SC Domain team deployer address on the output conduit
+        // during introductory phase. This allows the SC team to assist in the
+        // testing of a complete circuit. Once a broker dealer arrangement is
+        // established the deployer address should be `deny`ed on the conduit.
+        // Kissable(collateral.OUTPUT_CONDUIT).kiss(SC_DOMAIN_DEPLOYER_07);
+
+        // add RWA-00x contract to the changelog
+        DssExecLib.setChangelogAddress(collateral.gemID, collateral.GEM);
+        DssExecLib.setChangelogAddress(collateral.pipID, pip);
+        DssExecLib.setChangelogAddress(collateral.joinID, collateral.MCD_JOIN);
+        DssExecLib.setChangelogAddress(collateral.urnID, collateral.URN);
+        DssExecLib.setChangelogAddress(
+            collateral.inputConduitID, collateral.INPUT_CONDUIT
+        );
+        DssExecLib.setChangelogAddress(
+            collateral.outputConduitID, collateral.OUTPUT_CONDUIT
+        );
+
+        address ILK_REGISTRY = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+        IlkRegistryAbstract(ILK_REGISTRY).put(
+            collateral.ilk,
+            collateral.MCD_JOIN,
+            collateral.GEM,
+            DSTokenAbstract(collateral.GEM).decimals(),
+            3,
+            pip,
+            address(0),
+            collateral.ilkRegistryName,
+            collateral.ilk_string
+        );
+    }
+
+    function bytes32ToStr(bytes32 _bytes32) internal pure returns (string memory) {
+        return string(abi.encodePacked(_bytes32));
+    }
+
```

Comment:
```suggestion
```

Reply:
Added this at first to see if it fixed the issue, but somehow it didn't, which is why I decided to just add a string version of the ilk to the struct.